subworkflows/local/standardisation_profiles.nf

```diff
@@ -164,29 +169,31 @@ workflow STANDARDISATION_PROFILES {
     // Collect and replace id for db_name for prefix
     // Have to sort by size to ensure first file actually has hits otherwise
     // the script fails
-    ch_profiles_for_kraken2 = ch_input_profiles.kraken2
-                                .map {
-                                    meta, profiles ->
-                                        def new_meta = [:]
-                                        new_meta.tool = meta.tool == 'kraken2-bracken' ? 'kraken2' : meta.tool // replace to get the right output-format description
-                                        new_meta.id = meta.tool // append so to disambiguate when we have same databases for kraken2 step of bracken, with normal bracken
-                                        new_meta.db_name = meta.tool == 'kraken2-bracken' ? "${meta.db_name}-bracken" : "${meta.db_name}" // append so to disambiguate when we have same databases for kraken2 step of bracken, with normal bracken
-                                    [ new_meta, profiles ]
-                                }
-                                .groupTuple(sort: {-it.size()})
+    ch_profiles_for_kraken2 = groupProfiles(
+        ch_input_profiles.kraken2.map { meta, profile ->
+            def new_meta = [
+                // Replace the tool name to get the right output-format description.
+                tool: meta.tool == 'kraken2-bracken' ? 'kraken2' : meta.tool,
+                // Append so as to disambiguate when we have same databases
+                // for kraken2 step of bracken, with normal bracken.
+                id: meta.tool,
+                // Append so as to disambiguate when we have same databases
+                // for kraken2 step of bracken, with normal bracken.
+                db_name: meta.tool == 'kraken2-bracken' ? "${meta.db_name}-bracken" : "${meta.db_name}"
+
+            ]
+            return [meta + new_meta, profile]
+        },
+        [sort: { -it.size() }]
+    )
```

Comment:
It seems that the database name was not properly passed to the process `KRAKENTOOLS_COMBINEKREPORTS_KRAKEN`. 
The combined Bracken report is missing, and the combined Kraken2 report is missing the database name. Here are the files within the Kraken2 folder:
```
b1/
db2/
kraken2_null_combined_reports.txt
```

Reply:
I thought that was using the `id` value? Maybe needs both then, `id` and `db_name`.

Reply:
So indeed, the `KRAKENTOOLS_COMBINEKREPORTS_KRAKEN` process was configured to use the `db_name` key, rather than the `id` key. Both should be fixed now.